packages/stripe_platform_interface/lib/src/models/payment_sheet.dart

```diff
@@ -102,3 +102,30 @@ class PresentPaymentSheetParameters with _$PresentPaymentSheetParameters {
   factory PresentPaymentSheetParameters.fromJson(Map<String, dynamic> json) =>
       _$PresentPaymentSheetParametersFromJson(json);
 }
+
+@freezed
+class PaymentSheetResult with _$PaymentSheetResult {
```

Comment:
it seems to be a general error, not only targeted to payment sheet so I would advise to rename it 
https://github.com/stripe/stripe-react-native/blob/master/ios/StripeSdk.swift#L183

Reply:
yes I saw this as well am already fixing it into a class called `LocalizedErrorMessage`